newIDE/app/src/ObjectsList/index.js

```diff
@@ -750,6 +750,19 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
         selectObjectFolderOrObjectWithContext,
       ]
     );
+    React.useEffect(
```

Comment:
Could you regroup this effect with the other one setting the onDelete shortcut? Also, you can add a comment to the one above the instanciation of `keyboardShortcutsRef`